ttnn/cpp/ttnn/operations/data_movement/common/kernels/common.hpp

```diff
@@ -6,9 +6,45 @@
 // It's best to copy and paste the functions in rather than include the header as code size will likely explode
 // Best to separate in to cpp/hpp at some point to avoid the code size explosion but need to figure out the linking
 // issues
+#include <stdio.h>
+#include <cstring>
+#define MASK_64      0xFFFFFFFFFFFFFFC0
+#define OFFSET_64    0x000000000000003F
+#define MASK_16      0xFFFFFFFFFFFFFFF0
+#define OFFSET_16    0x000000000000000F
 
 namespace tt::data_movement::common {
 
+template <bool guaranteed_16B_alligned, bool copy_async>
+FORCE_INLINE
+void tt_memmove (
```

Comment:
This is great. Can you announce this utility function in TT developers. This might be useful for others.